modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -10363,28 +10366,33 @@ input SpectroscopyIntegrationTimeInput {
   # Observing wavelength.
   wavelength: WavelengthInput!
 
-  # Wavelength at which to measure Signal to Noise
-  signalToNoiseAt: WavelengthInput
-
   # Minimum desired signal-to-noise ratio.
   signalToNoise: SignalToNoise!
 
-  # Spatial profile PointSource/UniformSource/GaussianSource.
-  sourceProfile: SourceProfileInput!
+  # Wavelength at which to measure Signal to Noise
+  signalToNoiseAt: WavelengthInput
+
+  # Relevant data for targets in asterism.
+  asterism: [TargetInput!]!
 
   # Observing Band
   band: Band!
 
-  # Target radial velocity
-  radialVelocity: RadialVelocityInput!
```

Comment:
This is great, the previous version was clearly wrong taking a single radial velocity for all targets